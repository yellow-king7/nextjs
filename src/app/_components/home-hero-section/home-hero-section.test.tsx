import { render } from "@testing-library/react";
import { HomeHeroSection } from "./home-hero-section";

describe("Home hero section", () => {
  test("display the home hero section", () => {
    const { container } = render(<HomeHeroSection />);
    expect(container).toMatchSnapshot(`
    <section className="bg-hero-pattern bg-no-repeat bg-center h-80">
    <div className="flex flex-col-reverse xl:flex-row items-center">
      <div className="flex flex-col gap-5 mt-12 pb-5 text-center xl:text-right">
        <h3 className="text-xl dark:text-info xl:text-2xl ">
          خوش اومدی به ...
        </h3>
        <h1 className="text-3xl font-black gradient xl:text-5xl">
          مسیر صعود به قله های برنامه نویسی
        </h1>
        <p className="text-lg font-bold leading-8">
          هرجای مسیر برنامه نویسی که باشی با همراهی استاد های باتجربه کلاسبن
          میتونی بدون محدودیت به قله های بالاتر عود کنی. ما همیشه هواتو
          داریم.
        </p>
        <div className="mt-5 flex gap-4">
          <Button variant="primary" size="large">
            دوره های ری اکت و نکست
          </Button>
          <Button variant="neutral" size="large">
            مشاوره برنامه نویسی
          </Button>
        </div>
        <Image
          src="/images/frameworks.png"
          className="grayscale mt-4 opacity-70 m-auto xl:m-0"
          width={412}
          height={39}
          alt=""
        />
      </div>
      <Image
        src="/images/programmer-landing.svg"
        width={702}
        height={521}
        alt="classbon"
      />
    </div>
    <div className="container"></div>
  </section>`);
  });
});
