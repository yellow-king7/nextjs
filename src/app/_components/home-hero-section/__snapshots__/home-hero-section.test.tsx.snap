// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home hero section display the home hero section: 
    <section className="bg-hero-pattern bg-no-repeat bg-center h-80">
    <div className="flex flex-col-reverse xl:flex-row items-center">
      <div className="flex flex-col gap-5 mt-12 pb-5 text-center xl:text-right">
        <h3 className="text-xl dark:text-info xl:text-2xl ">
          خوش اومدی به ...
        </h3>
        <h1 className="text-3xl font-black gradient xl:text-5xl">
          مسیر صعود به قله های برنامه نویسی
        </h1>
        <p className="text-lg font-bold leading-8">
          هرجای مسیر برنامه نویسی که باشی با همراهی استاد های باتجربه کلاسبن
          میتونی بدون محدودیت به قله های بالاتر عود کنی. ما همیشه هواتو
          داریم.
        </p>
        <div className="mt-5 flex gap-4">
          <Button variant="primary" size="large">
            دوره های ری اکت و نکست
          </Button>
          <Button variant="neutral" size="large">
            مشاوره برنامه نویسی
          </Button>
        </div>
        <Image
          src="/images/frameworks.png"
          className="grayscale mt-4 opacity-70 m-auto xl:m-0"
          width={412}
          height={39}
          alt=""
        />
      </div>
      <Image
        src="/images/programmer-landing.svg"
        width={702}
        height={521}
        alt="classbon"
      />
    </div>
    <div className="container"></div>
  </section> 1`] = `
<div>
  <section
    class="bg-hero-pattern bg-no-repeat bg-center h-80"
  >
    <div
      class="flex flex-col-reverse xl:flex-row items-center"
    >
      <div
        class="flex flex-col gap-5 mt-12 pb-5 text-center xl:text-right"
      >
        <h3
          class="text-xl dark:text-info xl:text-2xl "
        >
          خوش اومدی به ...
        </h3>
        <h1
          class="text-3xl font-black gradient xl:text-5xl"
        >
          مسیر صعود به قله های برنامه نویسی
        </h1>
        <p
          class="text-lg font-bold leading-8"
        >
          هرجای مسیر برنامه نویسی که باشی با همراهی استاد های باتجربه کلاسبن میتونی بدون محدودیت به قله های بالاتر عود کنی. ما همیشه هواتو داریم.
        </p>
        <div
          class="mt-5 flex gap-4"
        >
          <button
            class="btn btn-primary btn-lg "
            type="button"
          >
            دوره های ری اکت و نکست
          </button>
          <button
            class="btn btn-neutral btn-lg "
            type="button"
          >
            مشاوره برنامه نویسی
          </button>
        </div>
        <img
          alt=""
          class="grayscale mt-4 opacity-70 m-auto xl:m-0"
          data-nimg="1"
          decoding="async"
          height="39"
          loading="lazy"
          src="/_next/image?url=%2Fimages%2Fframeworks.png&w=828&q=75"
          srcset="/_next/image?url=%2Fimages%2Fframeworks.png&w=640&q=75 1x, /_next/image?url=%2Fimages%2Fframeworks.png&w=828&q=75 2x"
          style="color: transparent;"
          width="412"
        />
      </div>
      <img
        alt="classbon"
        data-nimg="1"
        decoding="async"
        height="521"
        loading="lazy"
        src="/images/programmer-landing.svg"
        style="color: transparent;"
        width="702"
      />
    </div>
    <div
      class="container"
    />
  </section>
</div>
`;
